osm_to_sandbox/osm_to_sandbox.py

```diff
@@ -371,5 +395,41 @@ def main():
     print('Done.')
 
 
+def cli():
+    parser = argparse.ArgumentParser(
+        description="Downloads data from Overpass API and uploads it to the mapping "
+                    "sandbox.",
+        epilog="Because sandboxes are for grown-ups, too!",
+    )
+    parser = add_args(parser)
+    args = parser.parse_args()
+    args.bbox = [float(x.strip()) for x in args.bbox.split(',')]
```

Comment:
There's no need to modify the args object here. And it might backfire when `argparse` changes their return value. Could you please use a temporary variable here (e.g. `bbox`)?

Reply:
good spot. Updated in https://github.com/Zverik/osm_to_sandbox/commit/ba12e74e963b04d9aa18093cbd5e765cae4e7ac1